Extension/src/LanguageServer/client.ts

```diff
@@ -2234,6 +2244,27 @@ export class DefaultClient implements Client {
         return this.languageClient.sendRequest(IncludesRequest, params);
     }
 
+    public async getChatContext(token: vscode.CancellationToken): Promise<ChatContextResult> {
+        await withCancellation(this.ready, token);
+        let result: ChatContextResult;
+        try {
+            result = await this.languageClient.sendRequest(CppContextRequest, null, token);
+        } catch (e: any) {
+            // From <https://microsoft.github.io/language-server-protocol/specification#responseMessage>
+            // RequestCancelled = -32800, ServerCancelled = -32802,
+            if (e instanceof ResponseError && (e.code === -32800 /*RequestCancelled*/ || e.code === -32802 /*ServerCancelled*/)) {
```

Comment:
In multiple other places, we use constants for these (RequestCancelled, ServerCancelled).  Could you use those instead of hard-coding values?

Reply:
Of course, I am not that crazy to push this code further, I've done this because I could not find any ref to those and I was sure you were going to point me in the right direction. I think after I rebased onto main I can see them now :)

Reply:
fixed